crowd-server/crowd_server/apps/food_fact/views.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from .models import UserIn
+from .serializers import UserInSerialiser,UserOutSerialiser
+
+
+
+
+@api_view(['POST','GET'])
+def food_fact(resuest):
```

Comment:
This is wrong: when the HTTP method is `GET`, you query the data from the database and if the HTTP method is `POST`, you create a new record in the database.